packages/crud/src/crud/renderMetaReadOnlySection.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useQuery } from 'react-query'
+import { supabaseClient } from '@supabase/auth-helpers-nextjs'
+import { FormSectionReadOnlyStack } from '@gravis-os/form'
+import { Grid } from '@gravis-os/ui'
+
+const renderMetaReadOnlySection = ({ label, item }, userModuleTableName = "user") => {
+  const { data: creator } = useQuery({
+    queryFn: async () => {
+      const { data } = await supabaseClient
+        .from(userModuleTableName)
+        .select('full_name')
+        // @ts-ignore
+        .match({ id: item?.created_by })
+        .single()
+
+      return data
+    },
+    // @ts-ignore
+    queryKey: [userModuleTableName],
+    // @ts-ignore
+    enabled: Boolean(item?.created_by)
+  })
+
+  const { data: updater } = useQuery({
+    queryFn: async () => {
+      const { data } = await supabaseClient
+        .from(userModuleTableName)
+        .select('full_name')
+        // @ts-ignore
+        .match({ id: item?.updated_by })
+        .single()
+
+      return data
+    },
+    // @ts-ignore
+    queryKey: [userModuleTableName],
```

Comment:
Better to cast these instead of ts-ignoring. Can we fix them ts-ignores? Thanks!